gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+@Component({
+  selector: "app-expandable",
+  templateUrl: "./expandable.component.html",
+  styleUrls: ["./expandable.component.scss"],
+})
+export class ExpandableComponent implements OnInit {
```

Comment:
I think a more appropriate name for this could be chosen. The `Component` in a component class is generally used to indicate that it _is_ a component, not to describe what type of component it is. A more name that would fit convention better could be something like `SignalMessageComponent`.